mobile/ios/Podfile.lock

```diff
@@ -1,6 +1,4 @@
 PODS:
```

Comment:
🙃 You could have put this into a separate commit

Reply:
🙈